src/Validation/ConstraintViolationException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * This file is part of Lcobucci\JWT, a simple library to handle JWT and JWS
+ *
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ */
+
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Validation;
+
+use Lcobucci\JWT\Exception;
+
+/**
+ * @author Luís Otávio Cobucci Oblonczyk <lcobucci@gmail.com>
+ * @author Danny Dörfel <danny.dorfel@gmail.com>
+ * @author Marco Piveta <ocramius@gmail.com>
+ *
+ * @since 4.0.0
+ */
+final class ConstraintViolationException extends Exception
```

Comment:
`Exception` is `Lcobucci\JWT\Exception`... an initial work of #128.
Didn't want to come to a big inheritance tree now.